librz/include/rz_vector.h

```diff
@@ -68,11 +68,31 @@ RZ_API void rz_vector_fini(RzVector *vec);
 // frees the vector and calls vec->free on every element if set.
 RZ_API void rz_vector_free(RzVector *vec);
 
-RZ_API bool rz_vector_clone_into(
+RZ_API bool rz_vector_clone_intof(
 	RZ_NONNULL RZ_BORROW RZ_OUT RzVector *dst,
-	RZ_NONNULL RZ_BORROW RZ_IN RzVector *src);
+	RZ_NONNULL RZ_BORROW RZ_IN RzVector *src,
+	RZ_NULLABLE void (*item_cpy)(void *, void *));
 
-RZ_API RzVector *rz_vector_clone(RzVector *vec);
+RZ_API RZ_OWN RzVector *rz_vector_clonef(
+	RZ_NONNULL RZ_BORROW RZ_IN RzVector *vec,
+	RZ_NULLABLE void (*item_cpy)(void *, void *));
+
+static inline bool rz_vector_clone_into(
```

Comment:
@XVilka In fact, the original rz_vector_clone_into and rz_vector_clone were not removed, they were here

Reply:
But the API mark was removed still